mlflow/utils/request_utils.py

```diff
@@ -235,7 +235,7 @@ def cloud_storage_http_request(
     backoff_factor=2,
     backoff_jitter=1.0,
     retry_codes=_TRANSIENT_FAILURE_RESPONSE_CODES,
-    timeout=None,
+    timeout=(30, None),
```

Comment:
This seems reasonable. Let's test with a massive model using MPU (Mixtral) on s3 and ADLS to verify.